lib/menu.rb

```diff
@@ -0,0 +1,33 @@
+#This is the menu class
+
+class Menu
+
+attr_reader :menu, :starters, :mains, :desserts, :drinks
+
+  def initialize
+    
+    @starters = {
+      "Soup" => ["Soup",3.99],
+      "Salad" => ["Salad",4.99]
+    }
+    @mains = {
+      "Steak & Chips" => ["Steak & Chips",14.95],
+      "Burger & Chips" => ["Burger & Chips",11.95]
+    }
+    @desserts = {
+      "Tiramisu" => ["Tiramisu",6.50],
+      "Ice Cream" => ["Ice Cream",4.50]
+    }
+    @drinks = {
+      "Sparkling Water" => ["Sparkling Water",2.80],
+      "Still Water" => ["Still Water",2.80]
+    }
+    
+    @menu = [starters,mains,desserts,drinks]
+
+  end
+
+  def show
+    menu.each {|x| x.each{|x,y| puts "#{y.each {|x| x}}"}}
```

Comment:
Lint/ShadowingOuterLocalVariable: Shadowing outer local variable - `x`.<br>Lint/UnusedBlockArgument: Unused block argument - `x`. If it's necessary, use `_` or `_x` as an argument name to indicate that it won't be used.<br>Rails/Output: Do not write to stdout. Use Rails' logger if you want to log.